tests/integration/network/sepolia/conftest.py

```diff
@@ -14,6 +14,6 @@
 # run all tests with testnet
 @pytest.fixture(scope="module", autouse=True)
 def sepolia_env():
-    with boa.swap_env(boa.NetworkEnv(SEPOLIA_URI)):
+    with boa.set_network_env(SEPOLIA_URI):
```

Comment:
Shall we deprecate boa.swap_env?

Reply:
no, i think it can still be useful for more custom cases, e.g. https://github.com/vyperlang/titanoboa/blob/7271bf54e744848f6a25d1568803b28fcd2e5f3c/tests/integration/network/anvil/conftest.py#L79